x-pack/lib/monitoring/monitoring.rb

```diff
@@ -88,6 +88,14 @@ def config_collection?
       def get_binding
         binding
       end
+
+      def monitoring_endpoint
+        if LogStash::MonitoringExtension.use_direct_shipping?(LogStash::SETTINGS)
+          "/_bulk/monitoring"
```

Comment:
Is this correct? Testing this code against `7.6.0` generates an error, and glancing at the existing [beats code](https://github.com/elastic/beats/blob/a9aff6f1d8c0eff77b867a358f0fae0a641cbfb9/libbeat/monitoring/report/elasticsearch/client.go#L185) that does this, it appears to send to a bare `_bulk` endpoint (rather than `_bulk/monitoring`, and send a calculated index name with the request.